catalog/core/catalog-core-standardframework/src/test/java/ddf/catalog/resource/download/ReliableResourceDownloaderTest.java

```diff
@@ -61,6 +65,8 @@
 import ddf.catalog.resource.ResourceNotSupportedException;
 import ddf.catalog.resourceretriever.ResourceRetriever;
 
+@RunWith(PowerMockRunner.class)
```

Comment:
As far as I can tell, we're still getting coverage for `ReliableResourceDownloader`.